geth/common/types.go

```diff
@@ -171,9 +173,21 @@ type TxQueueManager interface {
 	DiscardTransactions(ids string) map[string]RawDiscardTransactionResult
 }
 
+// JailExecutor defines an interface which exposes method to be executed
+// against a Jail vm.
+type JailExecutor interface {
+	EvalExec(string) error
+	Run(string) (otto.Value, error)
+	Exec(string) (otto.Value, error)
```

Comment:
Hm, sounds reasonable. Could you add a comment about `Run` and `Exec` in this case so they're more distinguishable right from the interface?